earth_enterprise/BUILD_RHEL_CentOS.sh

```diff
@@ -0,0 +1,97 @@
+# check to make sure user is root
```

Comment:
I'm also thinking these scripts should be combined into 1. If someone wanted to download them to clone the repo and build, they'd have to get at least 2 files or download an archive and extract. Both of which include an extra step that we can eliminate.